starter-code/scripts/article.js

```diff
@@ -35,4 +54,6 @@ ourLocalData.forEach(function(ele) {
 
 articles.forEach(function(a){
   $('#articles').append(a.toHtml());
+  $('#author-filter').append(a.populateAuthorFilter());
+  $('#category-filter').append(a.populateCategoryFilter());
```

Comment:
You can do all the work of preventing duplicate appending here. Here is one way, and by doing this, you don't need to use Handlebars helpers. I think this looks a bit neater:
```
articles.forEach(function(a){
  $('#articles').append(a.toHtml());
  if ($('#author-filter').find(`option[value="${a.author}"]`).length == 0) {
    $('#author-filter').append(a.populateAuthorFilter());
  }
  if ($('#category-filter').find(`option[value="${a.category}"]`).length == 0) {
    $('#category-filter').append(a.populateCategoryFilter());
  }
});
```